app/src/main/java/co/nimblehq/compose/crypto/ui/screens/information/CoinInfoItem.kt

```diff
@@ -0,0 +1,108 @@
+package co.nimblehq.compose.crypto.ui.screens.information
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.constraintlayout.compose.Dimension
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.constraintlayout.compose.ConstraintLayout
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.screens.price.PriceChange
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp13
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp4
+import co.nimblehq.compose.crypto.ui.theme.Style
+import co.nimblehq.compose.crypto.ui.theme.Style.coinNameColor
+import co.nimblehq.compose.crypto.ui.theme.Style.textColor
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun CoinInfoItem(
+    modifier: Modifier,
+    // TODO Update values to Object on Integrate ticket
+    coinInfoItemTitle: String,
+    coinInfoItemPrice: String,
+    coinInfoItemPricePercent: Double
```

Comment:
Note: When we create the new model, we could remove `coinInfoItem` prefix to shorten the params name. The `CoinInfoItem` already shows the context 😄 

```suggestion
    title: String,
    price: String,
    priceChangePercent: Double
```